recipes/poppler/all/conanfile.py

```diff
@@ -100,9 +100,9 @@ def requirements(self):
         if self.options.get_safe("with_libiconv"):
             self.requires("libiconv/1.16")
         if self.options.fontconfiguration == "fontconfig":
-            self.require("fontconfig/2.13.92")
+            self.requires("fontconfig/2.13.92")
```

Comment:
I wonder why didn't it fail...

Reply:
juste because the default option values do not take this path